Cue/app/tabs/library/DeckView.js

```diff
@@ -92,6 +94,34 @@ class DeckView extends React.Component {
     this.props.flagCard(this.props.deckUuid, cardUuid, flag)
   }
 
+  _rateDeck = () => {
+    Alert.alert(
+      (Platform.OS === 'android'
+        ? 'Rate this deck'
+        : 'Rate This Deck'),
+      'Would you recommend this deck to others?',
+      [
+        {text: 'Cancel', style: 'destructive'},
+        {text: 'Would not Recommend', onPress: () => this.props.rateDeck(this.state.deck.uuid, -1)},
+        {text: 'Recommend',  onPress: () => this.props.rateDeck(this.state.deck.uuid, 1)},
```

Comment:
Same issue here as #162 — https://github.com/ClaytonPassmore/ProjectOrange/pull/162/files#diff-554e51b603c97565b81036119f86579fR135. You'll need to rearrange the buttons based on platform, though the Cancel button should stick to the bottom automatically on iOS if you specify `style: 'cancel'`.